### ISSUE TITLE
2 skids are missing

### ISSUE THREAD

**Issue by hoemotion** [Its-Vichy/lets-talk-about-discord#2]
[KeevoDev](https://github.com/KeevoDev/Nox-Mass-DM) (he skidded my shitty dpy mdm)
[notSocial404](https://github.com/notsocial404) (his old acc got banned he skidded many things one of the skidded things was my karuma dc nuker the only thing he added was a fucking token grabber i even have a screenshot but im too lazy for finding it he also made a yt tutorial about the tool but deleted it after me and the boys mass disliked it (yes the dislikes were still available back then lol))
There are a lot of yt skids but im too lazy for finding them 
I even saw people at fiverr skidding my stuff and the stuff by others and its kinda funny to see how brain f*cked they are but yeah thats it i will post here more skids if i find any)

**Comment by Its-Vichy** [comment]
social404 ask me to partner lol

**Comment by hoemotion** [comment]
> social404 ask me to partner lol

lmao

**Comment by 24hoster** [comment]
skidlist.txt completed

**Comment by Its-Vichy** [comment]
sure

**Comment by iGetUploaded** [comment]
> [KeevoDev](https://github.com/KeevoDev/Nox-Mass-DM) (he skidded my shitty dpy mdm) [notSocial404](https://github.com/notsocial404) (his old acc got banned he skidded many things one of the skidded things was my karuma dc nuker the only thing he added was a fucking token grabber i even have a screenshot but im too lazy for finding it he also made a yt tutorial about the tool but deleted it after me and the boys mass disliked it (yes the dislikes were still available back then lol)) There are a lot of yt skids but im too lazy for finding them I even saw people at fiverr skidding my stuff and the stuff by others and its kinda funny to see how brain f*cked they are but yeah thats it i will post here more skids if i find any)

Social Skidded SO MUCH SHIT I even called him out on and he denied it even when I had proof lmao

**Comment by Its-Vichy** [comment]
social 400 shit call me lot of time to made custom script to resell lmfao

**Comment by iGetUploaded** [comment]
lmao hes selling shit on Sellix wonder how much he skidded

**Comment by Scripped** [comment]
how do y'all know if someone is a skid? and it would be cool if someone could make a list of people

**Comment by Nostorian** [comment]
> how do y'all know if someone is a skid? and it would be cool if someone could make a list of people

its when the people who copy are so braindead that they dont even realize they are selling tools in an open srced platform (github) and saying we have to pay for it when the exact same tools are there for free by the original creators.

like i mean if you gonna skid shit from github dont be a retard and resell other's hardwork on the same website lmfao or any popular social media such as youtube

**Comment by Scripped** [comment]
damn that's sad... imagine scamming people out of something you didn't even make. it's so stupid.

**Comment by Scripped** [comment]
did @NotSocial404 delete everything? because i can't see any repos on his profile

**Comment by Nostorian** [comment]
> delete everything? because i can't see a

he aint got the balls to get termed again, so he doesnt skid to his github profile anymore, earlier he did in his old acc, but it got flagged coz for dmca shit and all and plagiarism

**Comment by Cipher-GF** [comment]
social404 :skull:

**Comment by Cipher-GF** [comment]
> damn that's sad... imagine scamming people out of something you didn't even make. it's so stupid.

social made so much cash out of skidding and making shit content

**Comment by Cipher-GF** [comment]
> did @NotSocial404 delete everything? because i can't see any repos on his profile

he claimed the username '@Social404' and is posting random shit and i've been forking all of it just in case ;)

btw @NotSocial404 is just his alt now or smth

**Comment by Nostorian** [comment]
> hit and i've been forking all of it just in case ;)

i am not going to fork it i am just importing his repos, incase he gets termed and shit or smthing worse happens where even forks are requested to be deleted as well we are screwed lol

**Comment by Cipher-GF** [comment]
cool

**Comment by Scripped** [comment]
apparently TheonlyIcebear is a skid as well

**Comment by Scripped** [comment]
and anytime someone points it out he just starts swearing at everyone... 🤷🏾‍♀️

**Comment by Scripped** [comment]
but im not sure, idk if he stole or not :/

**Comment by hoemotion** [comment]
He is not a skid, .gg/termed skidded his image logger code if im not wrong, but he might have become rude cuz you accused him accidentally?

**Comment by Scripped** [comment]
oh alright, tysm! alr im going to apologize to him then

**Comment by V4NSH4J** [comment]
> lmao hes selling shit on Sellix wonder how much he skidded 

Pretty much all of it. I called him out multiple times he just blames it on his developer and continues to skid more shit.

**Comment by hoemotion** [comment]
> > lmao hes selling shit on Sellix wonder how much he skidded
> 
> Pretty much all of it. I called him out multiple times he just blames it on his developer and continues to skid more shit.

lol

**Comment by Its-Vichy** [comment]
> Pretty much all of it. I called him out multiple times he just blames it on his developer and continues to skid more shit.

That why i never accept deal with social404

**Comment by bcky** [comment]
makes sense, the best tools always seem to be skidded

**Comment by Its-Vichy** [comment]
no

**Comment by Cipher-GF** [comment]
takes advantage of people

**Comment by hoemotion** [comment]
lol

**Comment by Cipher-GF** [comment]
and after he finds another tool he calls the ones he was reselling previously trash and garbage

**Comment by Cipher-GF** [comment]
i found that out from tokify.net

**Comment by Cipher-GF** [comment]
frfr :+1: :+1:

**Comment by Cipher-GF** [comment]
he is just lucky 

he blew up because of his trash videos on yt

**Comment by hoemotion** [comment]
Man I wish the police would arrest skids 💀

**Comment by Cipher-GF** [comment]
wait why is the FBI taking me away :scream: :cry: 

hold on, is my code 69% skidded ?? NO WAY !?!?

but seriously they need to arrest the skids who actually do it on purpose because they want to get profit and never give credits to the original author. Another reason they should be arrested is the fact that big skids claim they wrote 100% of the code and never knew that a code which was exact line by line ever existed :O

**Comment by hoemotion** [comment]
Yeah i mean i wouldnt care if a little function was „skidded“ but skidding a whole open source project and then selling it and telling the customers the code was written by himself is just a shame

**Comment by Cipher-GF** [comment]
fr
